### DISCUSSION TITLE
Headers in urql/core

### DISCUSSION THREAD

**Discussion by gabriiels** [urql-graphql/urql#2982] [Q&A] [answered]
Hello!

How are headers sent in urql/core?

Por example:
"Authorization":"Bearer ${authorizationToken}"
"example" : "123header"

**Comment by JoviDeCroock** [comment, marked as answer]
https://formidable.com/open-source/urql/docs/basics/core/#setting-up-the-client fetchOptions here, or you can use context.fetchOptions if you need it on query specific level

**Reply by gabriiels** [reply]
Thanks!
